utils/docker/Dockerfile.remote

```diff
@@ -1,25 +1,27 @@
+# To run this file execute:
+# docker build -f Dockerfile.remote . -t Lunarvim:remote
+
 FROM ubuntu:latest
 
-ARG NEOVIM_RELEASE=v0.5.0
+# Build argument to point to correct branch on GitHub
 ARG LVBRANCH=rolling
```

Comment:
Let's rename this to `LV_BRANCH` to align with the installer.

Reply:
Fixed in 7549c02